locales/da_DK/LC_MESSAGES/duckduckgo.po

```diff
@@ -1228,7 +1240,7 @@ msgstr "Foreslå DuckDuckGo en ny !Bang"
 
 #. Inside your browser title bar or tab title when visiting the <a href="https://duckduckgo.com/privacy.html">privacy page</a>.
 msgid "DuckDuckGo Privacy"
-msgstr ""
+msgstr "DuckDuckGo Privatlivs Politik"
```

Comment:
The original string is not for "DuckDuckGo Privacy Policy" ... though I'm not sure this token is used any more. We haven't done a very good job of cleaning them up.

Reply:
Hi @jbarrett thanks for your comment. However It's unclear to me if you like me to remove the string _OR_ ?

Could you please elborate and I'll fix it up as you would be requesting.

Reply:
Looking at this again, I think this translation is fine for where the token is used.